plotly/offline/offline.py

```diff
@@ -56,20 +58,40 @@ def init_notebook_mode():
         raise ImportError('`iplot` can only run inside an IPython Notebook.')
 
     global __PLOTLY_OFFLINE_INITIALIZED
-    # Inject plotly.js into the output cell
-    script_inject = (
-        ''
-        '<script type=\'text/javascript\'>'
-        'if(!window.Plotly){{'
-        'define(\'plotly\', function(require, exports, module) {{'
-        '{script}'
-        '}});'
-        'require([\'plotly\'], function(Plotly) {{'
-        'window.Plotly = Plotly;'
-        '}});'
-        '}}'
-        '</script>'
-        '').format(script=get_plotlyjs())
+    global __PLOTLY_USE_CDN
+
+    __PLOTLY_USE_CDN = connected
+
+    if connected:
+        # Inject plotly.js into the output cell
+        script_inject = (
+            ''
+            '<script>'
+            'requirejs.config({'
+            'paths: { '
+            '\'plotly\': [\'https://cdn.plot.ly/plotly-latest.min\']},'
```

Comment:
isn't this supposed to be `https://cdn.plot.ly/plotly-latest.min.js`? Are you missing the "`.js`"?